spec/models/drip_email_list_spec.rb

```diff
@@ -1,7 +1,7 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
 require 'rails_helper'
 
-RSpec.describe Dripemaillist, type: :model do
+RSpec.describe DripEmailList, type: => :model do
```

Comment:
The `=>` would not be proper.

```suggestion
RSpec.describe DripEmailList, type: :model do
```

Reply:
Yep! I noticed that when I reviewed my code again. Thank you!